vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -954,7 +975,33 @@ private void updateSelectedFragment(final MenuItem item) {
      */
     public void updateTabStyle(final int primaryColor, final int secondaryColor) {
         mToolbar.setBackgroundColor(primaryColor);
-        mFloatingActionButton.setBackgroundTintList(ColorStateList.valueOf(primaryColor));
+
+        AddFloatingActionButton menuButton = mFloatingActionsMenu.findViewById(com.getbase.floatingactionbutton.R.id.fab_expand_menu_button);
+
+        Class menuClass = FloatingActionsMenu.class;
+        try {
+            Field normal = menuClass.getDeclaredField("mAddButtonColorNormal");
```

Comment:
Are you obliged to do that? Isn't it possible using styles?

Reply:
Styles could be used but only for a one time setup.
Since the color changes at runtime, reflection was the easiest way.